### PR TITLE
New homepage design

### PR DESCRIPTION
**Pull request by patrickfeltes**
Adds new design for the homepage.

Currently missing:
1. Shadow colors (looking for ways to do this)
2. Countdown animation (coming in separate PR)
3. Fixing buildings to make them visible.

Screenshot:
homepage_new

### CODE REVIEW

File: app/src/main/java/org/hackillinois/android/view/home/CountdownManager.kt [hunk 1]
```diff
@@ -9,17 +9,17 @@ class CountdownManager(val listener: CountDownListener) {
 
     private val eventStartTime: Calendar = Calendar.getInstance().apply {
         timeZone = TimeZone.getTimeZone("America/Chicago")
-        set(2019, Calendar.FEBRUARY, 22, 16, 0, 0)
+        timeInMillis = 1582927200000
```
Comment (bstrauch24) [inline app/src/main/java/org/hackillinois/android/view/home/CountdownManager.kt]: Don't forget to use https://api.hackillinois.org/upload/blobstore/times/
I assume you're going to do that in another PR

File: app/src/main/java/org/hackillinois/android/view/home/HomeFragment.kt [hunk 2]
```diff
@@ -1,52 +1,79 @@
...
         super.onCreate(savedInstanceState)
 
+        context?.let {
+            val ongoingHeaderColor = ContextCompat.getColor(it, R.color.alternateTextColor)
+            ongoingEventsSection = EventsSection(
+                mutableListOf(),
+                "ONGOING",
+                ongoingHeaderColor,
+                this,
+                    it
```
Comment (bstrauch24) [inline app/src/main/java/org/hackillinois/android/view/home/HomeFragment.kt]: indentation

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -43,6 +43,11 @@
     <string name="close_text">CLOSE</string>
     <string name="logout_text">LOG OUT</string>
 
+    <string name="default_time_homepage">00</string>
```
Comment (bstrauch24) [inline app/src/main/res/values/strings.xml]: Let's make sure it says 00:00:00 at the end of the event this year haha

File: app/src/main/java/org/hackillinois/android/view/home/CountdownManager.kt (hunk 1 above)
Comment (patrickfeltes) [reply app/src/main/java/org/hackillinois/android/view/home/CountdownManager.kt]: Yep, that'll come soon.

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (patrickfeltes) [reply app/src/main/res/values/strings.xml]: lol yes
